crates/orchestrator/src/queue/sqs/mod.rs

```diff
@@ -54,6 +63,166 @@ impl QueueProvider for SqsQueue {
         let mut consumer = get_consumer(queue_url).await?;
         consumer.receive().await
     }
+
+    async fn create_and_setup_queues(&self) -> Result<()> {
+        // Setting up queues
+        let client = Self::create_sqs_client().await;
+        let dlq_arn = Self::create_dlq(&client).await?;
+        log::debug!("DLQ created. DLQ ARN: {}", dlq_arn);
+
+        for queue in Self::get_queue_configs() {
+            let redrive_policy = if queue.needs_dlq {
+                Some(format!(r#"{{"deadLetterTargetArn":"{}","maxReceiveCount":"{}"}}"#, dlq_arn, MAX_RECEIVE_COUNT))
+            } else {
+                None
+            };
+
+            let queue_url = Self::create_queue(&client, &queue.name, redrive_policy.as_deref()).await?;
+            log::info!("Created queue: {} at URL: {}", queue.name, queue_url);
+        }
+
+        // Setting up event bridge :
+        Self::setup_event_bridge().await?;
+
+        Ok(())
+    }
+}
+
+#[derive(Debug)]
+struct QueueConfig {
+    name: String,
+    needs_dlq: bool,
+}
+
+const VISIBILITY_TIMEOUT: i32 = 1800; // 30 minutes in seconds
+const DLQ_NAME: &str = "madara_orchestrator_job_handle_failure_queue";
+const MAX_RECEIVE_COUNT: &str = "5";
+
+impl SqsQueue {
+    async fn get_aws_configs() -> SdkConfig {
+        let region_provider = RegionProviderChain::default_provider();
+        from_env().region(region_provider).load().await
```

Comment:
from what i remember, we pass down AWS config normally from `config.rs`, can we not do something similar here?